apps/backoffice-v2/src/lib/blocks/variants/variant-checkers.ts

```diff
@@ -2,11 +2,12 @@ import { TWorkflowById } from '@/domains/workflows/fetchers';
 import { WorkflowDefinitionVariant } from '@ballerine/common';
 
 export const checkIsKybExampleVariant = (
-  workflowDefinition: Pick<TWorkflowById['workflowDefinition'], 'variant' | 'config' | 'version'>,
+  workflowDefinition?: TWorkflowById['workflowDefinition'],
```

Comment:
Revert. This function makes no sense with no arguments. If for some reason workflow definition is undefined it is a concern of the scope using the function and not the function itself.

Reply:
yeah but the usage of this makes it unnecessary hard.

I agree that providing all the attributes as the required in order to generate a proper signature for the method.
e.g. 
 {
  variant: (typeof WorkflowDefinitionVariant)[keyof typeof WorkflowDefinitionVariant];
  config: TWorkflowById['workflowDefinition']['config'];
  version: number;
}

and yet it's unnecessarily difficult to pick specific attributes for a helper method regardless of what attributes are being used for 

and yet i will revert it because it has no real meaning.